tests/functional/modules/test_zos_mvs_raw_func.py

```diff
@@ -1693,7 +1741,7 @@ def test_concatenation_fail_with_unsupported_dd_type(ansible_zos_module):
                             ),
                             dict(
                                 dd_data_set=dict(
-                                    data_set_name=DEFAULT_DATA_SET,
+                                    data_set_name="USER.PRIVATE.TEST",
```

Comment:
We don't have to figure this out now, but I would like to undertand why "USER.PRIVATE.TEST" remains hard coded ,  I may have to review the code.

Reply:
Hard but fixed

Reply:
but why is it hard to remove? what is the failure? what did you tried and why a new disposition on other parameter affects this?

Reply:
should be ok